comictaggerlib/autotagmatchwindow.py

```diff
@@ -258,4 +266,4 @@ def save_match(self) -> None:
                 )
                 break
 
-        ca.load_cache(list(metadata_styles))
+        ca.load_cache(list(metadata_styles))  # TODO Should this be what the others do in taggerwindow.py etc.?
```

Comment:
`ca.load_cache([*self.load_data_styles, *self.save_data_styles])` is done in `taggerwindow.py`. Should this be only the read styles or all the styles?

Reply:
For auto tagging it might be better to just clear the cache as it should provide at least a small speedup in processing